docker-compose/README.md

```diff
@@ -36,12 +36,14 @@ Setup can be easily bootstrapped by using the `<run-script>`. In order to do so
 
 ### Manual
 
-- Clone the [IoT Event Analytics](https://github.com/GENIVI/iot-event-analytics) repository into `<any folder>` using git<br>
-  Update the _.env_ file (IOTEA_PROJECT_DIR) with the absolute path to `<any folder>` - could be any valid local folder<br>
-  You can checkout a specific version tag if desired. __This version should match the SDKs you are copying in the next steps!__
-- Copy `<any folder>/src/sdk/javascript/lib/boschio.iotea-<version>.tgz` into _/src/edge.hal-interface-adapter_ AND _/docker-compose/talent_<br>
+- After cloning the vehicle-edge repository, execute `git submodule update --init --recursive` to get the contents of
+  [IoT Event Analytics](https://github.com/GENIVI/iot-event-analytics) repository as a submodule<br>
+  Update the _.env_ file (IOTEA_PROJECT_DIR) with the absolute path to `<vehicle-edge-dir>/iot-event-analytics`. 
+  IOTEA_PROJECT_DIR can also point to an externally cloned iot-event-analytics repository.<br>
+  A specific version tag can be checked out if desired. __This version should match the SDKs which are copied in the next steps!__
+- Copy `IOTEA_PROJECT_DIR/src/sdk/javascript/lib/boschio.iotea-<version>.tgz` into _/src/edge.hal-interface-adapter_ AND _/docker-compose/talent_<br>
   Update the _.env_ file (IOTEA_JS_SDK) with `boschio.iotea-<version>.tgz`
-- Copy `<any folder>/src/sdk/python/lib/boschio_iotea-<version>-py3-none-any.whl` into the _/src/edge.hal-interface_ directory<br>
+- Copy `IOTEA_PROJECT_DIR/src/sdk/python/lib/boschio_iotea-<version>-py3-none-any.whl` into the _/src/edge.hal-interface_ directory<br>
```

Comment:
If it is a submodule, why do we still have to copy something?

Reply:
We could directly install it from the libraries available in the submodule for Node.js. For Python it's sufficient to build/install the module directly from source without copying anything.